src/routes/safe/components/Apps/components/ConfirmTransactionModal.tsx

```diff
@@ -137,8 +138,6 @@ export const ConfirmTransactionModal = ({
   }
 
   const confirmTransactions = async () => {
-    const txData = encodeMultiSendCall(txs)
-
     await dispatch(
       createTransaction(
         {
```

Comment:
For `to` and `operation` here we need to make the same changes as mentined above